c-pallets/sminer/src/benchmarking.rs

```diff
@@ -115,8 +115,9 @@ benchmarks! {
         assert_eq!(1, reward_map.len());
         assert_eq!(reward_map[0].calculate_reward, 8000);
     }
-    
 
+		 // should depend also on CalculateRewardOrderMap total items, there is a long
+		 // for loop with a some read/writes
```

Comment:
I have changed to rely on CalculateRewardOrderMap total items for calculation:
https://github.com/CESSProject/cess/pull/88/files#diff-443a91eceace517ab334a2fff82de9b0e7340c3e743a28508bf4e7bd8cad0ce4R121-R138